### PR TITLE
Accuracy legend for RH colors

### PR DESCRIPTION
**Pull request by conbrad**
RH color legend based on: https://www.figma.com/file/2VuF3FNyhuf5gKD7jqEbeR/MoreCast?node-id=193%3A178
Screen Shot 2021-04-21 at 10 56 36 AM

https://wps-pr-902.apps.silver.devops.gov.bc.ca/morecast

### CODE REVIEW

**Review by Sybrand — APPROVED** [review]
Let's just wait for the accuracy ticket to go in 1st.

File: web/src/features/fireWeather/pages/MoreCastPage.tsx [hunk 1]
```diff
@@ -48,6 +49,11 @@ const useStyles = makeStyles(theme => ({
     display: 'flex',
     justifyContent: 'center',
     alignItems: 'center'
+  },
+  legend: {
+    display: 'flex',
+    alighnItems: 'flex-end',
```
Comment (andrea-williams) [inline web/src/features/fireWeather/pages/MoreCastPage.tsx:55]:
```suggestion
    alignItems: 'flex-end',
```

File: web/src/features/fireWeather/components/AccuracyColorLegend.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    'div:nth-child(3n)': {
+      marginRight: 0
+    }
+  },
+  green: {
+    backgroundColor: '#07A059',
+    borderTop: '2px solid #FFFFFF',
+    borderLeft: '2px solid #FFFFFF',
+    borderBottom: '2px solid #FFFFFF',
+    borderRadius: 2,
```
Comment (Sybrand) [inline web/src/features/fireWeather/components/AccuracyColorLegend.tsx]:
```suggestion
    borderRadius: '2px 0px 0px 2px',
```

File: web/src/features/fireWeather/components/AccuracyColorLegend.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+    borderBottom: '2px solid #FFFFFF',
+    height: 27,
+    width: 32
+  },
+  orange: {
+    backgroundColor: '#ED8001',
+    borderTop: '2px solid #FFFFFF',
+    borderRight: '2px solid #FFFFFF',
+    borderBottom: '2px solid #FFFFFF',
+    borderRadius: 2,
```
Comment (Sybrand) [inline web/src/features/fireWeather/components/AccuracyColorLegend.tsx]:
```suggestion
    borderRadius: '0px 2px 2px 0px',
```

**Review by andrea-williams — APPROVED** [review]

**Pull request merged**